pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -320,65 +317,58 @@ It should look similar to the below:
 
 ```bash
-./target/release/parachain-template-node build-spec --chain chain-spec.json --disable-default-bootnode --raw > chain-spec-raw.json
-```
-
-And re-generate the genesis state and wasm:
-
+pop build spec --chain chain-spec.json --disable-default-bootnode --genesis-state --genesis-code  
 ```
-./target/release/parachain-template-node export-genesis-head --chain chain-spec-raw.json > para-2000-genesis-state
+```bash
+┌   Pop CLI : Generate your chain spec
+│
+◇  An existing chain spec file is provided. Do you want to make additional changes to it?
+│  No 
 ```
 
-```
-./target/release/parachain-template-node export-genesis-wasm --chain chain-spec-raw.json > para-2000.wasm
-```
+> Pop CLI allows you to provide the path to an existing chain spec file to edit or regenerate the artifacts.
 
-We are now ready to run our parachain's collator node to sync with Paseo and start producing blocks.
+We are now ready to run to sync with Paseo Local and start producing blocks!
 
-## Running the Collator
+## Launch the Chain
 
-In order to run your parachain's collator node you will need the raw chain spec that our local Paseo network is using.&#x20;
+In order to run your parachain's collator you will need the raw chain spec that Paseo Local is using.
 
-This can be found in the output of when you ran the `pop up parachain -f network --verbose` command.:
+This can be found in the output when you ran the `pop up parachain -f network --verbose` command:
 
 <figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 12.20.38 PM (1).png" alt=""><figcaption></figcaption></figure>
 
-Copy this chain spec into our `my-parachain` directory:
+Copy this chain spec into our `my-chain` directory:
 
 ```bash
-cd my-parachain
+cd my-chain
 cp /var/folders/vl/txnq6gdj22s9rn296z0md27w0000gn/T/zombie-ddb5d2aa-704b-4658-af64-3cf9e3be5573/alice/cfg/paseo-local.json paseo-local-raw.json
 ```
 
-> Note: Your Paseo chain spec path will be different from the above.
+> Note: Your Paseo Local chain spec path will be different from the above.
+
+### Generate Node Key
 
 We will also need to create a node-key for your collator:
 
 ```bash
-cd my-parachain
-```
-
-```
-mkdir -p data/chains/my_parachain/network
-```
-
-```
-docker run -it parity/subkey:latest generate-node-key > ./data/chains/my_parachain/network/secret_ed25519
+cd my-chain
+mkdir -p data/chains/my_chain/network
+docker run -it parity/subkey:latest generate-node-key > ./data/chains/my_chain/network/secret_ed25519
 ```
 
 > Alternatively you can use Polkadot SDK binary instead of a Docker image:
->
 > ```
 > path/to/polkadot-sdk/target/debug/substrate-node key generate-node-key --file=secret_ed25519 --chain=./chain-spec-raw.json
 > ```
 >
-> <pre><code><strong>mv secret_ed25519 data/chains/my_parachain/network
+> <pre><code><strong>mv secret_ed25519 data/chains/my_chain/network
 > </strong></code></pre>
->
->
+
+### Run Collator
```

Comment:
Note that the above node key is placed in the `.data/` directory assuming that the user will run the collator copy/pasting the instructions.

More concretely the node key generated in the previous step will only be used if `--base-path ./data ` is provided.

Authorities need to be run with a specific node key. Otherwise `--unsafe-force-node-key-generation` can be provided to delegate the key generation to the node itself (this presents other nuances). But this is OK for local environments.

Reply:
Added a little `Note`